api/lib/crudOps.js

```diff
@@ -18,7 +18,7 @@ const getAll = async (res, query, name, ...args) => {
     }
     res.status(200).json(data);
   } catch (error) {
-    console.log(error, res, name, 'CRUD OPS ERROR, RES, NAME');
+    // console.log(error, res, name, 'CRUD OPS ERROR, RES, NAME');
```

Comment:
Can this console.log be removed?

Reply:
yes it can and it was removed in the latest commit